torch/csrc/distributed/c10d/ProcessGroupNCCL.hpp

```diff
@@ -642,6 +648,8 @@ class TORCH_API ProcessGroupNCCL : public Backend {
 
   void performNocolorSplit(at::Device device);
 
```

Comment:
nb: no longer modification to Work

Reply:
No, we have WorkNCCL and the timeout is actually added there...